### PR TITLE
Added 2 new entries for MPI

### PR DESCRIPTION
**Pull request by cschmitt-mpi**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by cschmitt-mpi** [comment]
Added two faculty members of MPIs.

File: csrankings.csv [hunk 1]
```diff
@@ -8726,6 +8727,7 @@ Meng Jiang 0001,University of Notre Dame,http://www.meng-jiang.com,LZIPfCkAAAAJ
 Mengchi Liu,Carleton University,http://www.scs.carleton.ca/~mengchi,NOSCHOLARPAGE
 Mengjie Zhang,Victoria University of Wellington,http://ecs.victoria.ac.nz/Main/MengjieZhang,hLvGrrkAAAAJ
 Merrick L. Furst,Georgia Institute of Technology,http://www.cc.gatech.edu/people/merrick-furst,NOSCHOLARPAGE
+Metin Sitti,Max Planck Institute,http://pi.is.mpg.de/person/sitti,YU4Ce_MAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:8730]: This person is not on this list, which I believe is canonical:

https://wp.mpi-sws.org/cis-copy-mpi-sws-org/people/

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (cschmitt-mpi) [reply csrankings.csv:8730]: This true, but we'll add him soon. But as you can see, he's listed here:https://www.is.mpg.de/person/sitti

By the way, the page is just a temporary page, the original one is: https://www.cis.mpg.de

Thanks,
CArina

Comment (emeryberger) [reply csrankings.csv:8730]: Can you go ahead and add him and let me know when this is done? Thanks.

Comment (cschmitt-mpi) [reply csrankings.csv:8730]: We updated the page:  https://www.cis.mpg.de and Metin Sitti is showing up there.
Hopefully, you can now confirm the changes.

Thanks,
CArina

Comment (emeryberger) [reply csrankings.csv:8730]: Yep, thanks!

**Pull request merged**